cluster-topology.md

```diff
@@ -0,0 +1,273 @@
+# Use Cluster API to change the Cluster Topology
+
+[In the previous walk-through](./your-first-cluster.md) we demonstrated how to create a cluster using a standard declarative configuration. Now we will show how to leverage this declarative configuration to update cluster topology with ease.
+
+**Table of Contents**
+
+- [Scale out control plane nodes](#scale-out-control-plane-nodes)
+  - [Scale from 1 to 3 control plane nodes](#scale-from-1-to-3-control-plane-nodes)
+- [More scale operations](#more-scale-operations)
+- [Add a new pool of worker nodes](#add-a-new-pool-of-worker-nodes)
+- [Remove a worker node](#remove-a-worker-node)
+- Next: [Changing Cluster Topology](#changing-cluster-topology)
```

Comment:
```suggestion
- Next: [Changing Cluster Topology](./cluster-topology.md)
```

Reply:
lol this is a bug, should be the "MachineHealthChecks and Remediation" stuff.

Reply:
We want nobody getting stuck in an infinite loop :D